src/components/Buttons/ClipboardButton.tsx

```diff
@@ -58,40 +64,55 @@ const ClipboardButton = ({ textToCopy, className }: ClipboardButtonProps) => {
       }, 3000)
     }
 
-    ReactTooltip.rebuild()
-
     return () => {
       if (interval) {
         clearInterval(interval)
       }
     }
   }, [hasBeenCopied, setSnackbarMessage, textToCopy, className, t])
 
-  if (!hasBeenCopied) {
-    return (
-      <div data-tip={t`Copy to clipboard`}>
-        <Clipboard
-          className={`${className} clipboard`}
-          size={15}
-          onClick={handleInput}
-          onKeyPress={handleInput}
-          role="button"
-          aria-label={t`Copy to clipboard`}
-          tabIndex={0}
-        />
-      </div>
-    )
-  } else {
-    return (
-      <div data-tip={t`Copied`}>
-        <Check className={`${className} check`} size={15} />
-      </div>
-    )
-  }
+  const clipboard = !hasBeenCopied ? (
+    <ClipboardWrapper data-tip={tipText ?? t`Copy to clipboard`}>
+      <Copy
+        className={`${className} clipboard`}
+        size={15}
+        onClick={handleInput}
+        onKeyPress={handleInput}
+        role="button"
+        aria-label={t`Copy to clipboard`}
+        tabIndex={0}
+      />
+    </ClipboardWrapper>
+  ) : (
+    <ClipboardWrapper className={className} data-tip={t`Copied`}>
+      <Check className={`${className} check`} size={15} />
+    </ClipboardWrapper>
+  )
+
+  return children || (Array.isArray(children) && children.length > 0) ? (
+    <div className={className}>
+      <CellChildren>{children}</CellChildren>
+      <CellClipboard>{clipboard}</CellClipboard>
+    </div>
+  ) : (
+    clipboard
+  )
```

Comment:
In which case will `children` be falsy AND `Array.isArray(children) && children.length > 0` be true? In other words, when will the condition be met?

Reply:
Never since `children.length > 0` excludes `[]` being a possibility

Reply:
I'm also confused by the question, the code is "is children truthy OR (is children a non-empty list)"...

Reply:
I changed it. Resolving.